Stack/Stack.cs

```diff
@@ -0,0 +1,31 @@
+﻿public class TStack<T> {

```

Comment:
![image](https://github.com/karthick-1205/Assignments/assets/75829713/b78ad861-0531-454f-b55e-2ac20c8d8c5b)
You have to declare TStack in a namespace and should use that in your test.
In Unittest1.cs, it should be referred with using.